src/patterns/dosdp-patterns/cellPartOfAnatomicalEntity.yaml

```diff
@@ -34,8 +39,9 @@ def:
   vars:
     - cell
     - anatomical_entity
-#  xrefs:
-#    value: pubs
+  annotations:
+    - annotationProperty: xref
+      value: pubs
```

Comment:
This  works, but can all be simplified by using the built-in xref key. (Earlier attempt at this seems to have been deleted).  Happy to revisit this in a second PR though.